page_templates/style_guide.md

```diff
@@ -0,0 +1,55 @@
+# Jina AI Documentation Style Guide
+
+Guides are publicly available and represent the Jina AI brand and the quality of the product. Please adhere to the following steps to ensure basic professionalism in the docs.
+
+## Guiding principles:
+
+-   **Accessibility**. Everyone who needs to should understand our documents.
+
+-   **Professional**. A consistent writing and formatting style will give a professional look to the documents.
+
+## Specific guidelines:
+
+**[ For anything not covered here, we default to the [Google developer style guide.](https://developers.google.com/style)  ]**
+
+1.  Check the spelling and grammar in your contributions. Most editors include a spell checker or have an available spell-checking plugin. You can also paste your text into a Google Doc or other document software for a more robust spelling and grammar check. We use American English spelling.
+
+2.  Avoid run-on sentences. Generally, keep sentences to less than 13 words longs.
+
+3.  When explaining technical steps, use a numbered step by step process. Don't list out every step in a single paragraph.
+
+4.  When explaining general concepts, paragraphs should be used. Yet paragraphs should not be over six sentences long.
+
+5.  Use the second person tense and describe actions to the user. (this is the same as active style or imperative style)
+
+> You should open Jina using python.
+> In your pod, you will find.
+
+6.  When using Jina terms, they should be Capitalized. Flow, Pod, JinaD. Anything found in the Jina glossary or Jina 101. Including Executor types (Evaluator, Segmenter, etc.)
+
+7. Don't capitalize Flow types (index, query). Don't capitalize the adjective or  `-tion`  form (i.e. evaluation, evaluating, segmenting, etc)
+
+8.  Use YAML, not yaml.
+
+9.  Do not use compound sentences, chains of clauses, and location-specific idioms, making text hard to understand and translate.
+
+10.  Avoid semicolons.
+
+11.  Don’t use abbreviations without explaining them first.
```

Comment:
How far do we take this? Do we assume knowledge of REST, gRPC, etc? ID??

Reply:
I'm not trying to be pedantic ;) but sooner or later someone is going to try to reverse engineer ID and that won't be fun for anyone involved

Reply:
No that's a very good point! I've updated that point with some guidelines from [apple](**https://developer.apple.com/library/archive/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/APIAbbreviations.html**). In general I think we'll have to play these things by ear and fall back on our copy writer. But let me know what you think